biogeochem/FatesPatchMod.F90

```diff
@@ -169,10 +170,12 @@ module FatesPatchMod
     !---------------------------------------------------------------------------
     
    ! DISTURBANCE 
-    real(r8) :: disturbance_rates(n_dist_types) ! disturbance rate [0-1/day] from 1) mortality 
-                                                !                                 2) fire 
-                                                !                                 3) logging mortatliy
-    real(r8) :: fract_ldist_not_harvested       ! fraction of logged area that is canopy trees that weren't harvested [0-1]
+    real(r8) :: disturbance_rates(n_dist_types)           ! disturbance rate [0-1/day] from 1) mortality
+                                                          !                                 2) fire
+                                                          !                                 3) logging mortatliy
+                                                          !                                 4) land use change
+    real(r8) :: landuse_transition_rates(n_landuse_cats)  ! land use tranision rate
```

Comment:
ideally these should be dynamically allocated based on the number of cats defined in the dataset, or perhaps the number of cats that we want to remember based on the combining or grouping of cats in the dataset.